packages/core/utils/src/modules-sdk/medusa-internal-service.ts

```diff
@@ -252,12 +252,22 @@ export function MedusaInternalService<
       input: any | any[] | SelectorAndData | SelectorAndData[],
       @MedusaContext() sharedContext: Context = {}
     ): Promise<InferEntityType<TEntity> | InferEntityType<TEntity>[]> {
-      if (!isDefined(input) || (Array.isArray(input) && input.length === 0)) {
+      if (
```

Comment:
can we use `isPresent` here?

Reply:
yes you are right 👍 much better, I ll add that in